.travis.yml

```diff
@@ -34,27 +34,51 @@ dist: trusty
 language: python
 git:
   depth: 1000
-before_install: 
+before_install:
 addons:
+  hostname: mtt_host
   apt:
     packages:
       - doxygen
       - doxygen-doc
       - doxygen-latex
       - doxygen-gui
       - graphviz
+
+      # install MPI libs
+      - openmpi-bin
+      - mpich
+      - libmpich-dev
+      - libopenmpi-dev
+
+      # install SLURM
```

Comment:
Why install slurm?

Reply:
I installed SLURM to test if I could run the bat that tested the multi-node capabilities without spinning up a cluster. I forgot to remove all the libraries I installed when testing out the other bat. I'll remove these since they are not necessary for the single node bat to execute.